brew/build.gradle

```diff
@@ -21,4 +21,8 @@ dependencies {
     // because they are used for code generation types.
     compile 'junit:junit:4.12'
     compile 'org.junit.jupiter:junit-jupiter-api:5.1.1'
+
+
+    compile 'mysql:mysql-connector-java:8.0.8-dmr'
```

Comment:
The `brew` layer itself should not have a dependency on the connectors. It's the job of a higher layer to sort that out.

Reply:
I should remove this, this was for testing purposes